components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx

```diff
@@ -156,7 +156,7 @@ export function AllowAddChangeNetworkPanel(props: Props) {
   return (
     <StyledWrapper>
       <CenterColumn>
-        <FavIcon src={`chrome://favicon/size/64@1x/${originInfo.originSpec}`} />
+        <FavIcon src={`chrome://favicon/size/64@1x/${encodeURIComponent(originInfo.originSpec)}`} />
```

Comment:
I just tested this - `chrome://favicon` doesn't work with URL encoded origins (it expects it to be in the `https://domain.tld` format) so I think we should revert this one.